yarn.lock

```diff
@@ -4229,6 +4229,11 @@
   "resolved" "https://registry.npmjs.org/dedent/-/dedent-0.7.0.tgz"
   "version" "0.7.0"
 
+"deep-diff@^0.3.5":
```

Comment:
- [ ] Kindly update the readme to match [this template](https://github.com/microverseinc/readme-template).